itl/command-plugins.conf

```diff
@@ -1696,6 +1696,7 @@ object CheckCommand "by_ssh" {
 	}
 
 	vars.by_ssh_address = "$check_address$"
+	vars.by_ssh_port = "$ssh_port$"
```

Comment:
You are right, this could be the solution to _my_ case... But wouldn't it be good if similar check commands behaved similarly? 
[Like the ssh CheckCommand for example...](https://github.com/oliverrahner/icinga2/blob/024f405426dd9224a772fbbb1cc889e7d8cf20f4/itl/command-plugins.conf#L956-L962
)